src/main/java/com/minecolonies/inventory/InventoryField.java

```diff
@@ -0,0 +1,177 @@
+package com.minecolonies.inventory;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.nbt.NBTTagList;
+import net.minecraftforge.common.util.Constants;
+
+/**
+ * The custom chest of the field.
+ */
+public class InventoryField extends InventoryCitizen
+{
+    /**
+     * NBTTag to store the slot.
+     */
+    private static final String TAG_SLOT = "slot";
+
+    /**
+     * NBTTag to store the items.
+     */
+    private static final String TAG_ITEMS = "items";
+
+    /**
+     * NBTTag to store the custom name.
+     */
+    private static final String TAG_CUSTOM_NAME = "name";
+
+    /**
+     * NBTTag to store the inventory.
+     */
+    private static final String TAG_INVENTORY = "inventory";
+
+    /**
+     * Returned slot if no slat has been found.
+     */
+    private static final int NO_SLOT = -1;
+
+    /**
+     * The inventory stack.
+     */
+    private ItemStack[] stackResult   = new ItemStack[1];
+
+    /**
+     * The custom name of the inventory.
+     */
+    private String customName         = "";
+
+    /**
+     * Creates the inventory of the citizen.
+     *
+     * @param title         Title of the inventory.
+     * @param localeEnabled Boolean whether the inventory has a custom name.
+     */
+    public InventoryField(final String title, final boolean localeEnabled)
+    {
+        super(title, localeEnabled);
+        customName = title;
+    }
+
+    @Override
+    public int getSizeInventory()
+    {
+        return 1;
+    }
+
+    @Override
+    public int getInventoryStackLimit()
+    {
+        return 1;
+    }
+
+    @Override
+    public int getHotbarSize()
+    {
+        return 0;
+    }
+
+    /**
+     * Getter for the stack in the inventory. Since there is only one slot return always that one.
+     * @param index the slot.
+     * @return the itemStack in it.
+     */
+    public ItemStack getStackInSlot(int index)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1161)